lib/services.cf

```diff
@@ -307,6 +307,9 @@ body action fresh_systemd_state
     ifelapsed => "0";
 @endif
 
+    background => "false"; # Needed for versions 3.15.x and older that did not
+                           # support empty bodies. When 3.15.x is no longer
+                           # supported, this can be removed.
```

Comment:
I am 90% sure you can just add 
```
any::
```

To make the body not empty.

Reply:
ACK, adjusted.